### ISSUE TITLE
IP Addresses for shaping and cleaning up data analysis

### ISSUE THREAD

**Issue by BigActual** [JacksonCollege/jc-theme-2015#259]
@banksashleym would you please write to IT and ask them for some IP related information? I'd like to isolate data for your internal network in a new view and I'd like to use a filter to remove your staff's activity from our analysis. Would you please work with IT to determine
1) The range of IP addresses students use when they are on the school's system?
2) If there is a separate network or range of IP addresses for staff, that would be useful to break out as well
3) Do the machines you, Dotty, and the other marketing staff work on have static IP Addresses? If so, what are they?

**Comment by banksashleym** [comment]
Definitely. I have emailed our IT department with these questions and will get back to you with the IP addresses when I receive them!

**Comment by banksashleym** [comment]
Here is what I received from IT:
The range of our internal IP addresses all fit into this range 172.16.0.0/12

**Comment by BigActual** [comment]
Could you please send back a request for more detail? If I exclude that entire range, I exclude all student and staff traffic from analytics. That would make analytics data useless. Could you ask them for the following specific ranges or actual addresses?

- What are the IP addresses for the computers in the Marketing department (This one is most important. You guys are on the site all day and skew your own data)?
- What is the IP range of staff computers?
- Is there a separate staff wifi? If so, what is that range?
- What is the IP range for the student accessible wifi systems and for the student accessible computers in the labs and the libraries?

We will use this data to shape content availability and to improve the user experience. So I hope they understand the value this information will provide and could spend some time on these details? Here's an example. By far, email is the number one tool used on the website. But is it really? Is it just that staff spend all day on that page, so they skew our data?

The answer may be that IT can't figure it out in this much detail, but let's see if they have it? It may change everything about the way we perceive the site's usefulness.
